documentation/userguide/docs/redshift_datasets.md

```diff
@@ -0,0 +1,151 @@
+
+**Redshift Dataset fields**
+
+| Field                      | Description                                                       | Required | Editable |Example
+|----------------------------|-------------------------------------------------------------------|----------|----------|-------------
+| Redshift Connection        | Name of the Redshift connection used to read the Redshift tables  | Yes      | No       | main-cluster-userA
+| Redshift database schema   | Name of the Redshift schema where the tables are stored           | Yes      | No       | public
+| Redshift tables            | List of tables to be imported. They can be added at a later stage | No       | Yes      | customer, orders
+
+Once a Redshift dataset has been imported, the dataset and its imported tables can be searched by any user in the Catalog. 
+
+## :material-card-search-outline: **Navigate dataset tabs**
+**Overview**
+
+This tab includes meaningful metadata about the dataset and the Redshift connection used.
+
+![](pictures/redshift_datasets/redshift_dataset_overview.png#zoom#shadow)
+
+**Data**
+
+This tab shows the Redshift database, schema and tables imported. From here we can add, edit, delete and see the details of a table.
+
+![](pictures/redshift_datasets/redshift_dataset_data.png#zoom#shadow)
+
+**Shares**
+Show a list of the share requests for this Dataset. It is possible to verify the health and reapply shares for the
+entire Dataset
+
+### **Manage Redshift Tables**
+
+**Add tables**
+
+![](pictures/redshift_datasets/redshift_add_tables.png#zoom#shadow)
+
+**View and edit tables**
+
+We can view the schema of a table directly from the Data tab, by clicking on the **Open table schema** button.
+
+![](pictures/redshift_datasets/redshift_schema.png#zoom#shadow)
+
+We can also see a full view of the table by selecting the arrow in the Actions column. A new window for the table
+will open. In this view we can edit the metadata of the table in data.all (Tags, glossary, description) and we can
+see the schema in full-width in the Columns tab.
+
+![](pictures/redshift_datasets/redshift_table.png#zoom#shadow)
+
+**Delete a table**
+
+We can delete Redshift tables by clicking on the trash icon next to the table we want to "un-import". Un-import is a better
+word to describe what will happen: the metadata of the table will be deleted from data.all Catalog, but the original
+Redshift table still exists in Redshift.
+
+## :material-pencil-outline: **Edit and update a dataset**
+Data owners can edit the dataset by clicking on the **edit** button, editing the editable fields and saving the changes.
+
+## :material-trash-can-outline: **Delete a dataset**
+To delete a dataset, in the selected dataset window click on the **delete** button in the top-right corner. data.all Redshift 
+Datasets don't deploy any CloudFormation stack, no additional resources need to be cleaned up. The original Redshift tables
+will still exist in Redshift.
```

Comment:
Is there any stop gaps on if existing shares on dataset we cannot delete? If so lets add a call out here